delphi/app/pki_model.py

```diff
@@ -46,12 +44,12 @@ def update_room(self, node, new_count):
         :param node: the name of the room to update
         :param new_count: the new people count for that room
         """
-        if node == 's1':
-            self.room_counts['s1'] = new_count
-        elif node == 's2':
-            self.room_counts['s2'] = new_count
-        elif node == 's3':
-            self.room_counts['s3'] = new_count
+        if node == '252':
+            self.room_counts['252'] = new_count
+        elif node == '256':
+            self.room_counts['256'] = new_count
+        elif node == '260':
+            self.room_counts['260'] = new_count
```

Comment:
why not opt for something like:
```python
if node in self.room_counts:
    self.room_counts[node] = new_count
```
is it necessary to hard code this?

Reply:
lol no sorry my brain is fried obviously. Will change this